sdk/monitor/monitor-opentelemetry-exporter/src/export/trace.ts

```diff
@@ -29,6 +32,10 @@ export class AzureMonitorTraceExporter implements SpanExporter {
 
   private readonly _options: AzureExporterInternalConfig;
 
+  /**
+   * Initializes a new instance of the AzureMonitorTraceExporter class.
+   * @param AzureExporterConfig - Exporter configuration.
```

Comment:
Might be worth mentioning that if they specify _nothing_ then the default behavior is...probably to pull from the environment? (not sure what your default is)